test/Fuzzing/fuzzer_test.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-build-swift -parse-as-library -sanitize=fuzzer %s -o %t
+// RUN: not %t -only_ascii=1 -max_len=3 | %FileCheck %s
+// CHECK: Crash!
```

Comment:
Do these tests also need `XFAIL: linux`? Or `REQUIRES: libfuzzer` or something, so that Darwin builds without compiler-rt can still pass.